copter/source/docs/mission-command-list.rst

```diff
@@ -292,12 +292,13 @@ OR the change in heading (in degrees) when "rel/abs" field is "1"
 **Sec** - not supported.  meant to limit the rotation speed (in deg/sec)
```

Comment:
This is now supported. It does set the maximum rotation speed provided it is not larger than the limits set by the aircraft.